main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -621,4 +629,68 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
         }
     }
   }
+
+  /**
+    * Package the current project and release it on GitHub.
+    */
+  def release(flix: Flix): Validation[Unit, BootstrapError] = {
+    val formatter = flix.getFormatter
+
+    // Ensure that we have a manifest
+    val manifest = optManifest match {
+      case Some(m) => m
+      case None => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingManifest))
+    }
+
+    // Check if `github` option is present
+    val githubRepo = manifest.repository match {
+      case Some(r) => r
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.NoLinkedProject))
```

Comment:
Sometimes you linebreak before return and other times you dont. Make a choice :)